source/stdafx.h

```diff
@@ -46,15 +46,9 @@ static std::string GetGameDirectory() // already stored in Filepath_Game
 
 static std::string GetUserDirectory() // already stored in Filepath_User
 {
-    static const char* GTA_User_Dir_Path = (char*)0x00C92368; // SA 1.0 US
-    static const auto GTA_InitUserDirectories = (char* (__cdecl*)())0x00744FB0; // SA 1.0 US
+    static const auto GTA_InitUserDirectories = (char* (__cdecl*)())0x00744FB0; // SA 1.0 US - CFileMgr::InitUserDirectories
```

Comment:
can we use version manager here? with all these addresses it will be hard to support other games/versions in the future

Reply:
how will it work in III/VC? Separate build?